### PR TITLE
tools/dm: refine dashboard description

### PR DESCRIPTION
**Pull request by csuzhangxc**
- refine the description for task's dashboard
- add a description for instances' dashboard

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @lilin90 PTAL

File: tools/dm/monitor.md [hunk 1]
```diff
@@ -12,13 +12,30 @@ category: tools
 
 ## Task
 
+Grafana dashboard 默认名称为 `DM`。
```
[the remark this answers is not in the document]
Comment (CaitinChen) [reply tools/dm/monitor.md]: If you use "Grafana dashboard 默认名称为 `DM`", users might think it means "the default name of Grafana dashboard is `DM`".

File: tools/dm/monitor.md [hunk 2]
```diff
@@ -12,13 +12,30 @@ category: tools
 
 ## Task
 
+Grafana dashboard 默认名称为 `DM`。 
+
+### overview
+
+> overview 下包含运行当前选定 task 的所有 DM-worker instances 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。
```

File: tools/dm/monitor.md [hunk 3]
```diff
@@ -12,13 +12,30 @@ category: tools
...
+
+| metric 名称 | 说明 | 告警说明 |
+|:----|:------------|:----|
+| task state | 同步子任务的状态 | N/A |
+| storage capacity | relay log 占有的磁盘的总容量  | N/A |
+| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
+| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |
+| load progress | loader 导入过程的进度百分比，值变化范围为：0 %- 100 %  | N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

File: tools/dm/monitor.md [hunk 4]
```diff
@@ -12,13 +12,30 @@ category: tools
...
+| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数。| N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```

File: tools/dm/monitor.md [hunk 5]
```diff
@@ -35,15 +52,15 @@ category: tools
 | write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒。| N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]: Please help remove this period. Thanks.

File: tools/dm/monitor.md [hunk 6]
```diff
@@ -76,3 +93,36 @@ category: tools
 | execution latency | syncer 执行 transaction 到下游的耗时，单位：秒 | N/A |
 | unsynced tables | 当前子任务内还未收到 shard DDL 的分表数量 | N/A |
 | shard lock resolving | 当前子任务是否正在等待 shard DDL 同步，大于 0 表示正在等待同步 | N/A |
+
+
+## Instances
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
## Instance
```

Comment (CaitinChen) [reply tools/dm/monitor.md]: or "instance"?

File: tools/dm/monitor.md [hunk 7]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+
+Grafana dashboard 默认名称为 `DM-instances`。
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
在 Grafana dashboard 中，instance 的默认名称为 `DM-instances`。
```

File: tools/dm/monitor.md [hunk 8]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+
+> 当前默认告警规则在 [Task](#task) 部分已有说明，此处不再重复。
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
当前默认告警规则参见 [Task](#task) 部分。
```

File: tools/dm/monitor.md [hunk 9]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+
+### Relay log
+
+| metric 名称 | 说明 | 告警说明 |
+|:----|:------------|:----|
+| storage capacity | relay log 占有的磁盘的总容量  | N/A |
+| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
+| process exits with error | relay log 在 DM-worker 内部遇到错误并且退出了  | N/A |
+| relay log data corruption | relay log 文件损坏的个数 | N/A |
+| fail to read binlog from master | relay 从上游的 mysql 读取 binlog 时遇到的错误数 | N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| fail to read binlog from master | relay 从上游的 MySQL 读取 binlog 时遇到的错误数 | N/A |
```

File: tools/dm/monitor.md [hunk 10]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+| fail to write relay log | relay 写 binlog 到磁盘时遇到的错误数 | N/A |
+| binlog file index | relay log 最大的文件序列号。如 value = 1 表示 relay-log.000001 | N/A |
+| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |
+| binlog pos | relay log 最新文件的写入 offset  | N/A |
+| read binlog duration | relay log 从上游的 MySQL 读取 binlog 的时延，单位：秒 |  N/A |
+| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒。| N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒 | N/A |
```

File: tools/dm/monitor.md [hunk 11]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+| binlog size | relay log 写到磁盘的单条 binlog 的大小 | N/A |
+
+### task
+
+| metric 名称 | 说明 | 告警说明 |
+|:----|:------------|:----|
+| task state | 同步子任务的状态 | N/A |
+| load progress | loader 导入过程的进度百分比，值变化范围为：0 %- 100 %  | N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

File: tools/dm/monitor.md [hunk 12]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数。| N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```

File: tools/dm/monitor.md (hunk 5 above)
Comment (csuzhangxc) [reply tools/dm/monitor.md]: removed, thanks.

File: tools/dm/monitor.md [hunk 13]
```diff
@@ -12,52 +12,69 @@ category: tools
...
 
-## Loader
+### Loader
 
 下面 metrics 仅在 `task-mode` 为 `full` 或者 `all` 模式下会有值。
 
 | metric 名称 | 说明 | 告警说明 |
 |:----|:------------|:----|
-| load progress | loader 导入过程的进度百分比，值变化范围为：0 %- 100 %  | N/A |
+| load progress | loader 导入过程的进度百分比，值变化范围为：0%- 100%  | N/A |
```
Comment (CaitinChen) [inline tools/dm/monitor.md]:
```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

**Comment by CaitinChen** [comment]
@GregoryIan PTAL

File: tools/dm/monitor.md [hunk 14]
```diff
@@ -12,52 +12,69 @@ category: tools
...
+overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。
+
+| metric 名称 | 说明 | 告警说明 |
+|:----|:------------|:----|
+| task state | 同步子任务的状态 | N/A |
+| storage capacity | relay log 占有的磁盘的总容量  | N/A |
+| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
+| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |
+| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
+| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```
Comment (GregoryIan) [inline tools/dm/monitor.md]:
```suggestion
| binlog file gap between master and syncer | 与上游 master 相比 binlog replication 落后的 binlog file 个数 | N/A |
```

File: tools/dm/monitor.md [hunk 15]
```diff
@@ -12,52 +12,69 @@ category: tools
 
 ## Task
 
+在 Grafana dashboard 中，DM 默认名称为 `DM`。
```
Comment (GregoryIan) [inline tools/dm/monitor.md]: is `DM-task` better?

File: tools/dm/monitor.md [hunk 16]
```diff
@@ -76,3 +93,36 @@ category: tools
 | execution latency | syncer 执行 transaction 到下游的耗时，单位：秒 | N/A |
 | unsynced tables | 当前子任务内还未收到 shard DDL 的分表数量 | N/A |
 | shard lock resolving | 当前子任务是否正在等待 shard DDL 同步，大于 0 表示正在等待同步 | N/A |
+
+
+## Instance
+
+在 Grafana dashboard 中，instance 的默认名称为 `DM-instances`。
+
+当前默认告警规则参见 [Task](#task) 部分。
```
Comment (GregoryIan) [inline tools/dm/monitor.md]: 这个表述少了点什么吗？

File: tools/dm/monitor.md (hunk 15 above)
Comment (csuzhangxc) [reply tools/dm/monitor.md]: sounds good, then change the dashboard in DM-Ansible?

File: tools/dm/monitor.md [hunk 17]
```diff
@@ -76,3 +93,36 @@ category: tools
...
+| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒 | N/A |
+| binlog size | relay log 写到磁盘的单条 binlog 的大小 | N/A |
+
+### task
+
+| metric 名称 | 说明 | 告警说明 |
+|:----|:------------|:----|
+| task state | 同步子任务的状态 | N/A |
+| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
+| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```
Comment (GregoryIan) [inline tools/dm/monitor.md]: ditto with L28

**Comment by GregoryIan** [comment]
Rest LGTM

File: tools/dm/monitor.md (hunk 15 above)
Comment (csuzhangxc) [reply tools/dm/monitor.md]: changed

File: tools/dm/monitor.md (hunk 16 above)
Comment (csuzhangxc) [reply tools/dm/monitor.md]: add more descriptions.

File: tools/dm/monitor.md (hunk 17 above)
Comment (csuzhangxc) [reply tools/dm/monitor.md]: addressed.

**Pull request merged**